interface/src/Application.cpp

```diff
@@ -1972,7 +1973,7 @@ void Application::initializeUi() {
     rootContext->setContextProperty("Quat", new Quat());
     rootContext->setContextProperty("Vec3", new Vec3());
     rootContext->setContextProperty("Uuid", new ScriptUUID());
-    rootContext->setContextProperty("Assets", new AssetMappingsScriptingInterface());
+    rootContext->setContextProperty("Assets", DependencyManager::get<AssetMappingsScriptingInterface>().data());
 
```

Comment:
What is the reason for changing AssestMappingsScriptingInterface to a singleton? I afraid that this change might have unpredicted consequences sown the road